01-shapefile-attributes.Rmd

```diff
@@ -294,71 +284,78 @@ levels(lines_HARV$TYPE)
 summary(lines_HARV$TYPE)
 
 #plot the lines data, apply a diff color to each category
-plot(lines_HARV, col=lines_HARV$TYPE,
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
```

Comment:
Given four levels of TYPE why it is only plotting in two colors?  This makes interpreting the figure, especially in next section when adding line width, challenging.